samigo/samigo-app/src/webapp/jsf/evaluation/detailedStatistics.jsp

```diff
@@ -35,6 +35,12 @@ $Id: histogramScores.jsp 38982 2007-12-06 13:05:38Z gopal.ramasammycook@gmail.co
 **********************************************************************************/
 --%>
 -->
+<script>
+  $(document).ready(function(){
+    $('#histogram\\:detailedStatisticsMenuLink').addClass('current');

```

Comment:
can you add this class via JSF attribute?

Reply:
You got me!

I tried many ways, via includes and via beans, didn't find a way to do it without using facelets :( That's why I used Javascript. Do you know a way to do it via pure mojarra JSF?

Reply:
it's not just a simple styleClass?

Reply:
The problem is that the JSF template should tell the JSF menu template which tab is the current one, and the "include" directive we're using doesn't allow parameters. That's why I added a JS snippet to apply the current class to the tab, the template tells the menu where the user is.

If you know an alternative approach I'll be glad to apply it!

Thanks for the review @ottenhoff